src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
@markcowl I know I'm seeing this on 4.6.0, but I don't have the source for that, so don't know if that's changed since.